ql/src/test/queries/clientpositive/ctas_direct.q

```diff
@@ -0,0 +1,118 @@
+-- SORT_QUERY_RESULTS
+--! qt:dataset:alltypesorc
+
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.acid.direct.insert.enabled=true;
+set hive.exec.max.dynamic.partitions=200;
+set hive.exec.max.dynamic.partitions.pernode=200;
+
+DROP TABLE IF EXISTS source;
+
+DROP TABLE IF EXISTS test_orc_ctas;
+
+DROP TABLE IF EXISTS test_orc_mmctas;
+
+DROP TABLE IF EXISTS test_parquet_mmctas;
+
+DROP TABLE IF EXISTS test_avro_mmctas;
+
+DROP TABLE IF EXISTS test_textfile_mmctas;
+
+DROP TABLE IF EXISTS test_partition_orc_ctas;
+
+DROP TABLE IF EXISTS test_partition_orc_mmctas;
+
+DROP TABLE IF EXISTS test_partition_parquet_mmctas;
+
+DROP TABLE IF EXISTS test_partition_avro_mmctas;
+
+DROP TABLE IF EXISTS test_partition_textfile_mmctas;
+
+CREATE TABLE IF NOT EXISTS source STORED AS ORC TBLPROPERTIES('transactional'='true') AS (SELECT cint, cfloat, cdouble, cstring1, ctimestamp1 FROM alltypesorc);
```

Comment:
You have dropped the table in this test. If there is an issue and it still exists then we have an issue 😄 
Also `IF EXISTS` might hide issues if we forget to drop a table in advance.

Reply:
I was thinking about the `CREATE TABLE IF NOT EXISTS`.
I would keep the `DROP TABLE IF EXISTS`, and only use `CREATE TABLE ....` instead

Reply:
Oh ok. I will make the changes for this.